marvel-app/features/search-page/SearchPageViewModelEvents.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  SearchPageViewModelEvents.swift
+//  marvel-app
+//
+//  Created by Michal on 08/08/2022.
+//
+
+import Foundation
+
+protocol SearchPageViewModelEvents: AnyObject {
+    func comicsFetched(comics: ComicDataWrapper)
+    func nothingFound(notFound: Bool)
```

Comment:
Function notingFound() suggests that I will never find comics. Please change to something like hasFoundComics()